README.md

```diff
@@ -37,22 +37,64 @@ $ psql -U postgres -c "create user dpr_user password 'secret' createdb;"
 $ psql -U postgres -c "create database dpr_db owner=dpr_user;"
 
 # create tables
-python manager.py db upgrade
+$ python manager.py createdb
+$ python manager.py populate
 ```
 
+While development, if you make changes to database structure, Eg: added new table,
+renamed column etc... You will have to **drop** all the tables and recreate them
+
+```
+$ python manager.py dropdb
+$ python manager.py createdb
+$ python manager.py populate
+```
+
+Note: Be careful! Doing so **all** of you data will be erased. **Never, ever** run
```

Comment:
The sentence is confusing, probably this is better:
Executing commands above would erase **all** the data (or database).